src/templates/blog-post.tsx

```diff
@@ -91,8 +95,13 @@ const BlogHeader = ({ readingTime, frontmatter }) => {
 const BlogArticleTemplate: React.FC<BlogArticleTemplateProps> = ({
   data,
   location,
+  pageContext,
 }) => {
   const markdown = data.markdownRemark;
+  const breadcrumb = [
+    ...pageContext.breadcrumb,
+    { pathname: location.pathname, label: markdown.frontmatter.title },
+  ];
```

Comment:
What's the benefit of having parts of the breadcrumb in the page context? I suggest to move the breadcrumb "calculation" completely to the template and remove it from the `create-pages` script. I also recommend to use `t(...)` wherever possible (in this case for "Blog"), see other comment for my reasoning.
```suggestion
  const breadcrumb = [
    { pathname: '/', label: 'Satellytes' },
    { pathname: '/blog', label: t('navigation.blog') },
    { pathname: location.pathname, label: markdown.frontmatter.title },
  ];
```

(Disclaimer: `t` is not yet available in this context so if you accept this suggestion please make sure to also get it with `useTranslation()`.)

Reply:
I wrote the breadcrumbs in the page context because I wanted to keep the structure where all the data comes from the create-pages files. 
But I agree with you and I think that the advantage of using `t()` is much bigger and that in other places in the code the data doesn't always come from the create-pages files either. I have now adapted the code according to your review and it looks better than before. Thanks for your feedback 😊